src/osm_configurator/view/toplevelframes/calculation_frame.py

```diff
@@ -1,29 +1,32 @@
 from __future__ import annotations
```

Comment:
a comment for this file in general, you might have forgotten the start caluclation button, you just simply start it after pressing one of the buttons to set where the calculation starts